core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThread.java

```diff
@@ -95,6 +99,32 @@ public KafkaStoreReaderThread(String bootstrapBrokers,
             org.apache.kafka.common.serialization.ByteArrayDeserializer.class);
     consumerProps.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
             org.apache.kafka.common.serialization.ByteArrayDeserializer.class);
+
+    consumerProps.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG,
+            config.getString(SchemaRegistryConfig.KAFKASTORE_SECURITY_PROTOCOL_CONFIG));
+    if (config.getString(SchemaRegistryConfig.KAFKASTORE_SECURITY_PROTOCOL_CONFIG).equals(
```

Comment:
@hachikuji, no. I'll make a helper method. Good catch -- newb mistake!

@Ishiihara, each config option has a default value, which I believe means the String key will never be missing from the config. Unless I'm missing something?